test/clientWithSignupsDisabled.test.ts

```diff
@@ -21,3 +28,105 @@ test('signUp()', async () => {
   expect(data).toBeNull()
   expect(user).toBeNull()
 })
+
+test('generateLink() should be able to generate multiple links', async () => {
+  const invitedUser = faker.internet.email().toLowerCase()
+  const { error, data: firstInvite } = await authAdmin.generateLink('invite', invitedUser, {
+    redirectTo: 'http://localhost:9997',
+  })
```

Comment:
```
const { error, data: firstInvite } = await authAdmin
  .generateLink('invite', 
  'email@example.com', 
  {
    redirectTo: 'http://localhost:9997',
  })
```

If there are no SMTP settings on GoTrue, this will not return an action_link (we are investigating)